Content.Server/Chat/Systems/SpeakOnTriggerSystem.cs

```diff
@@ -0,0 +1,42 @@
+using Content.Server.Explosion.EntitySystems;
+using Content.Shared.Timing;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.Chat.Systems;
+
+public sealed class SpeakOnTriggerSystem : EntitySystem
```

Comment:
Add some documentation here

Reply:
I did it in the component! That's usually how I see it done (At least for this it would be copy pasted the same from the component). I feel like it should either be in one or the other and I guess I feel like component makes more sense. Let me know what you think!

Reply:
I mean, it doesn't hurt but won't be a blocker.
Usually the components have the documentation since one system can handle multiple components.